source/nodejs/tsconfig.json

```diff
@@ -1,10 +1,10 @@
 {
 	"compilerOptions": {
 		"module": "commonjs",
-		"target": "es5",
+		"target": "es2017",
 		"lib": [
 			"dom",
-			"es2015"
+			"es2017"
```

Comment:
was there a reason we needed to upgrade here? I am all for it but just want to make sure this will not break existing assumptions/code/users etc.

Reply:
Previously it was to use the object.entries function to simply be able to go through key value pairs in an object. Now I have switched the code to do this in a easier fashion with an explicit for loop, so this upgrade is no longer necessary